nvflare/private/fed/server/server_command_agent.py

```diff
@@ -82,11 +83,14 @@ def execute_command(self, request: CellMessage) -> CellMessage:
         else:
             return make_reply(ReturnCode.INVALID_REQUEST, "No server command found", fobs.dumps(None))
 
-    def _get_client(self, token):
+    def _get_client(self, client_name):
         fl_server = self.engine.server
         client_manager = fl_server.client_manager
         clients = client_manager.clients
-        return clients.get(token)
+        for _, client in clients.items():
+            if client_name == client.name:
+                return client
+        return None
```

Comment:
Why can't we use token in this scenario?

Reply:
The client restarted and got a new token.